ocs_ci/deployment/openstack.py

```diff
@@ -0,0 +1,13 @@
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class OPENSTACKIPI:
+    """
+    A class to handle OpenStack IPI specific deployment
+    """
+
+    def __init__(self):
+        logger.info("OPENSTACK IPI")
+        # TODO: OCP,OCS deployment
```

Comment:
I take it this is just setting up the ground work for openstack IPI deployments. Should we expect the actual OCP/OCS deployment code in this PR or in a follow up?

Reply:
That may be addressed in a follow up PR. This is being added just to get the Tier1 tests to run.

Reply:
If that is the case then we may want to log that the deployment isn't actually happening, or raise an error here saying it isn't supported yet. That way we avoid confusion from anyone trying to deploy a cluster with openstack IPI.